ledger-core/application/benchs/cloud_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package benchs
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/insolar/assured-ledger/ledger-core/application/testutils/launchnet"
+	"github.com/insolar/assured-ledger/ledger-core/instrumentation/inslogger/instestlogger"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
+)
+
+func Benchmark_MultiPulseOnCloud_Timed(b *testing.B) {
+	instestlogger.SetTestOutput(b)
+
+	for numNodes := 2; numNodes <= 5; numNodes++ {
+		b.Run(fmt.Sprintf("Nodes %d", numNodes), func(b *testing.B) {
+			teardown, cloudRunner, err := runCloud(numNodes)
+			if err != nil {
+				b.Fatal("network run failed")
+			}
+
+			runner := &benchRunner{
+				benchLimiter: &timedBenchLimiter{pulseTime: int(cloudRunner.ConfProvider.PulsarConfig.Pulsar.PulseTime)},
+				n:            100,
+			}
+
+			apiAddrs := []string{}
+			for _, el := range cloudRunner.ConfProvider.GetAppConfigs() {
+				apiAddrs = append(apiAddrs, el.TestWalletAPI.Address)
+			}
+
+			run := runner.runBenchOnNetwork(b, numNodes)
+			retCode := run(apiAddrs)
+			if retCode != 0 {
+				b.Fatal("failed test run")
+			}
+
+			teardown()
+		})
+	}
+}
+
+// TODO: Stop pulsar after network startup.
+// This functionality should be done in PLAT-782
+func Benchmark_SinglePulseOnCloud_N(b *testing.B) {
```

Comment:
With current approach global log output is writing a lot to stdout. So it is really hard to distinguish go bench output from logs. Bench output should go to different file if we want to see benchmark results